Octokit.Reactive/Clients/ObservableAuthorizationsClient.cs

```diff
@@ -46,6 +46,57 @@ public IObservable<Authorization> Get(int id)
         }
 
         /// <summary>
+        /// Creates a new personal token for the authenticated user.
+        /// </summary>
+        /// <remarks>
+        /// This method requires authentication.
+        /// See the <a href="https://developer.github.com/v3/oauth_authorizations/#create-a-new-authorization">API documentation</a> for more information.
+        /// </remarks>
+        /// <param name="newAuthorization">Describes the new authorization to create</param>
+        /// <exception cref="AuthorizationException">
+        /// Thrown when the current user does not have permission to make this request.
+        /// </exception>
+        /// <exception cref="TwoFactorRequiredException">
+        /// Thrown when the current account has two-factor authentication enabled and an authentication code is required.
+        /// </exception>
+        /// <exception cref="ApiException">Thrown when a general API error occurs.</exception>
+        /// <returns>The created <see cref="Authorization"/>.</returns>
+        public IObservable<ApplicationAuthorization> Create(NewAuthorization newAuthorization)
+        {
+            Ensure.ArgumentNotNull(newAuthorization, "authorization");
+
+            return _client.Create(newAuthorization).ToObservable();
+
```

Comment:
This PR is perfect so I'm going to nitpick on the extraneous line endings between the `return` statement and closing brace. :stuck_out_tongue: